src/sentry/api/endpoints/organization_member_team_details.py

```diff
@@ -67,6 +67,40 @@ def _can_access(self, request, member, organization):
 
         return False
 
+    def _can_admin_team(self, request, organization, team_slug):
+        acting_member = OrganizationMember.objects.get(
+            organization=organization,
+            user__id=request.user.id,
+            user__is_active=True,
+        )
```

Comment:
Will this query fail when the current user is a super user, or do super users never get this far?

Reply:
So superusers should not get this far since `_can_access` checks for it and it would take precedence and short circuit.

Hmm, this would be a problem here, but it seems to lineup to the previous behavior irt superusers: https://github.com/getsentry/sentry/blob/3fe226da57ed734f8dbcd359a96f89528f66b2a2/src/sentry/api/endpoints/organization_member_team_details.py#L155